gulpfile.js

```diff
@@ -38,7 +38,8 @@ const watchedSources = [
 ];
 
 const scripts = [
-    'src/terminateProcess.sh'
+    'src/terminateProcess.sh',
```

Comment:
Is "teminatePRocess.sh" still needed?

Reply:
Yes as far as I know. It's mac and Linux only

Reply:
I can't find `src/terminateProcess.sh` in the code base. How does it get injected for max and Linux cases?

Reply:
It's here: https://github.com/Microsoft/vscode-node-debug2/blob/master/src/terminateProcess.sh#L1